scripts/ci/sync_release_assets.py

```diff
@@ -177,13 +186,20 @@ def main() -> None:
     parser.add_argument("--no-rerun-cli", action="store_true", help="Don't upload CLI")
     args = parser.parse_args()
 
+    # Wait for a bit before doing anything.
+    # Useful on CI because the google-cloud-storage upload action doesn't guarantee to read-your-writes.
+    wait_time_secs = float(args.wait)
+    if wait_time_secs > 0.0:
+        print(f"Waiting for {wait_time_secs}s…")
+        time.sleep(wait_time_secs)
+
     gh = Github(args.github_token, timeout=args.github_timeout)
     repo = gh.get_repo(args.github_repository)
-    release = repo.get_release(args.github_release)
+    release = cast(GitRelease, get_any_release(repo, args.github_release))
```

Comment:
That `cast` should probably a check of `is not None` instead, as this is a possible outcome of `get_any_release`

Reply:
Yes, but I just want it to crash anyway in that case, so it seemed redundant to manually check for null just to throw the same exception myself :shrug: